tests/integration_tests/test_event_loop_integration.py

```diff
@@ -0,0 +1,70 @@
+import json
+import textwrap
+from click.testing import CliRunner
+from exasol_advanced_analytics_framework import deploy
+from tests.test_package.test_event_handlers.event_handler_test import \
+    FINAL_RESULT
+from tests.utils.parameters import bucketfs_params, db_params
+
+
+def _create_schema(db_conn, schema):
+    db_conn.execute(f"DROP SCHEMA IF EXISTS {schema} CASCADE;")
+    db_conn.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
+
+
+def _deploy_scripts(language_alias, schema_name):
+    args_list = [
+        "scripts",
+        "--dsn", db_params.address(),
+        "--user", db_params.user,
+        "--pass", db_params.password,
+        "--schema", schema_name,
+        "--language-alias", language_alias,
+        "--develop"
+    ]
+    runner = CliRunner()
+    result = runner.invoke(deploy.main, args_list)
+    assert result.exit_code == 0
```

Comment:
I suggest you are using this instead of the cli

https://github.com/exasol/advanced-analytics-framework/blob/78cb9c92fa905132c346d289623598d39def480c/exasol_advanced_analytics_framework/deployment/scripts_deployer.py#L11